lib/fluent/plugin/formatter_csv.rb

```diff
@@ -35,6 +36,10 @@ class CsvFormatter < Formatter
       config_param :fields, :array, value_type: :string
       config_param :add_newline, :bool, default: true
 
+      def thread_key
+        "#{plugin_id}_csv"
```

Comment:
```suggestion
        "#{owner.plugin_id}_csv_formatter_#{usage}_csv"
```

Formatter plugins (owned plugins) are unique to `owner.plugin_id` and `usage`.